commands/preview.js

```diff
@@ -1,106 +1,48 @@
+ * a given bundle. The site URL and bundle URL should be passed in. Upon completion, waitUntilMobified is called, to be sure that the
+ * adaptation is complete.
 *
- * If `site.json` does not exist, this command will just go to the specified URL.
- *
- * ```
- *    this.demoTest = function (client) {
- *      browser.preview();
- *    };
- * ```
- * or with a URL
- *
+ * Usage:
  * ```
  *    this.demoTest = function (client) {
- *      browser.preview('http://my-awesome-project.com');
+ *      browser.preview('https://www.merlinspotions.com', 'https://localhost:8443/loader.js');
  *    };
  * ```
  *
- * @method attributeEquals
- * @param {string} [URL] (optional) The URL to be previewed.
- * @param {function} callback The function to be called on completion.
- * @api assertions
+ * @method preview
+ * @param {string} [url] Corresponds to the Site URL field on https://preview.mobify.com
+ * @param {string} [bundle] Corresponds to the Bundle Location field on https://preview.mobify.com
+ * @param {function} [callback] Optional callback function to be called when the command finishes.
+ * @api commands
  */
 
-var path = require('path');
 var qs = require('querystring');
 
-exports.command = function(url, callback) {
+exports.command = function(url, bundle, callback) {
     var browser = this;
 
-    try {
-        var siteConfig = require(path.join(path.resolve('./'), '/tests/system/site.json'));
-    } catch (e) {
-        if (e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
-            console.log('Not using optional site.json. Looking for site.js...');
-        }
-    }
-
-    try {
-        var siteConfig = require(path.join(path.resolve('./'), '/tests/system/site.js'));
-    } catch (e) {
-        if (e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
-            console.log('Not using optional /tests/system/site.js.');
-        }
+    if (arguments.length < 2) {
+        throw new Error('Usage: browser.preview(url, bundle, callback)');
     }
 
-    try {
-        var siteConfig = require(path.join(path.resolve('./'), '/system/site.js'));
-    } catch (e) {
-        if (e instanceof Error && e.code === 'MODULE_NOT_FOUND') {
-            console.log('Not using optional /system/site.js.');
-        }
+    if (typeof bundle === 'function') {
+        callback = bundle;
+        bundle = '';
     }
 
-    if (siteConfig) {
-        var site = siteConfig.profiles[siteConfig.activeProfile];
+    var bundleUrl = bundle || 'https://localhost:8443/loader.js';
 
-        if (typeof url === 'function') {
-            callback = url;
-            url = site.siteUrl;
-        }
+    var params = qs.stringify({'url': url, 'site_folder': bundleUrl});
 
-        // First checks for the URL, otherwise uses the site.siteURL, then makes sure
-        // that there is an http prefix. The preview function doesn't need this, but
-        // the browser.get() method does.
-        url = url || site.siteUrl;
-
-        if (!url.match(/^http/)) {
```

Comment:
This is fine. `url` gets passed as a param to preview.mobify.com, which doesn't require the URL to begin with `HTTP`.